examples/ansible/setup_libvirt_host.yml

```diff
@@ -25,36 +29,35 @@
         name: libvirtd
         wait_for:
           path: /var/run/libvirt/libvirt-sock
-
   tasks:
-
     - name: Retrieve image for workload {{ workload.name }}
       become: true
       containers.podman.podman_image:
         name: "{{ workload.image }}"
         state: present
       register: retrieve_workload_image
+      failed_when: retrieve_workload_image is failed
 
     - name: Install tools for workload {{ workload.name }}
       become: true
       ansible.builtin.command: >-
         podman container runlabel {{ workload.labels.install }} {{ workload.image }}
       register: workload_runlabel_install
-      changed_when:
-        - ('already exist' not in workload_runlabel_install.stdout)
+      changed_when: "'already exist' not in workload_runlabel_install.stdout"
```

Comment:
This is an unnecessary formatting/whitespace change to the existing code, and necessitated the additional double quoting of the condition which was already validly using parentheses wrapping to avoid complaints about a condition entry starting with a quote character.